syntax = "proto3";
package test;

message EchoReq
{
	string message = 1;
};

message EchoRes
{
	string message = 1;	
};

service EchoService
{
	rpc Echo (EchoReq) returns (EchoRes) {}	
};